README.md

```diff
@@ -1,128 +1,9 @@
-*Russian version available [here](README_ru.md)*.
+Button control modified to use only one button per 1 blind
```

Comment:
you're rewriting this file instead of adding info about new possibilities. You shouldn't remove information about already implemented features.